codersdk/templates.go

```diff
@@ -72,11 +74,12 @@ type UpdateTemplateACL struct {
 }
 
 type UpdateTemplateMeta struct {
-	Name             string `json:"name,omitempty" validate:"omitempty,template_name"`
-	DisplayName      string `json:"display_name,omitempty" validate:"omitempty,template_display_name"`
-	Description      string `json:"description,omitempty"`
-	Icon             string `json:"icon,omitempty"`
-	DefaultTTLMillis int64  `json:"default_ttl_ms,omitempty"`
+	Name                         string `json:"name,omitempty" validate:"omitempty,template_name"`
+	DisplayName                  string `json:"display_name,omitempty" validate:"omitempty,template_display_name"`
+	Description                  string `json:"description,omitempty"`
+	Icon                         string `json:"icon,omitempty"`
+	DefaultTTLMillis             int64  `json:"default_ttl_ms,omitempty"`
+	AllowUserCancelWorkspaceJobs bool   `json:"allow_user_cancel_workspace_jobs"`
```

Comment:
Should this be `*bool` as well so the value doesn’t need to be provided every request?

Reply:
Frankly speaking, I would leave it as is and fix it as part of https://github.com/coder/coder/issues/5066. This is the outcome of API design. If I fix it here, it will be confusing for customers (optional params vs `""`).